lib/client.js

```diff
@@ -25,6 +25,24 @@ var ipSchema = Joi.string().ip({
   cidr: false
 });
 
+var emailSchema = Joi.string().email();
+
+var validate = function(type, obj) {
+  if (type === 'ip') {
+    if (Joi.validate(obj, ipSchema).error !== null) {
```

Comment:
Will this work with IP6?

Reply:
No, currently only IPv4 is supported.

Reply:
We have a next up item for IP6 support in iprepd, should we make sure this works here now? Also fine if we do it in another issue and can create an issue to track it.

Reply:
Yeah, I think we should do it in another issue (created https://github.com/mozilla-services/ip-reputation-js-client/issues/42)

Reply:
Sounds good 👍